lib/web_ui/lib/src/engine/platform_dispatcher.dart

```diff
@@ -531,6 +530,21 @@ class EnginePlatformDispatcher extends ui.PlatformDispatcher {
         textEditing.channel.handleTextInput(data, callback);
         return;
 
+      case 'flutter/contextmenu':
```

Comment:
I've omitted any web or browser namespacing (not "flutter/webcontextmenu") because other methods don't seem to include platform names like that.  And maybe in the future this could be used on other platforms.

In the framework PR I'll make the API specific (BrowserContextMenu.disable or something like that).